tests/plugins/test_store_metadata.py

```diff
@@ -15,26 +15,11 @@
 from copy import deepcopy
 from textwrap import dedent
 
-try:
-    import koji as koji
-except ImportError:
-    import inspect
-    import os
-    import sys
-
-    # Find our mocked koji module
-    import tests.koji as koji
-    mock_koji_path = os.path.dirname(inspect.getfile(koji.ClientSession))
-    if mock_koji_path not in sys.path:
-        sys.path.append(os.path.dirname(mock_koji_path))
-finally:
-    del koji
-    from atomic_reactor.plugins.exit_koji_promote import KojiPromotePlugin
```

Comment:
Actually I think we need this part. The reason it's there is that the `koji` module is not yet available for Python 3, but we still want to test our own koji-using code under Python 3.

To this end, we have a pretend koji module: https://github.com/projectatomic/atomic-reactor/blob/master/tests/koji/__init__.py

The `ImportError` stuff above is to handle this case, and allow our pretend koji module to be used when testing a module that uses `import koji`.

Yes, this should probably have had a big comment explaining that.